### ISSUE TITLE
Install Failure: card-mod-ace_editor requires card-mod-script (= 0.11.0)

### ISSUE THREAD

**Issue by bertdeb** [decko-commons/decko#608]
```
gem install decko

ERROR:  Error installing decko:
	card-mod-ace_editor requires card-mod-script (= 0.11.0)
```

This is happening from a brand new Ubuntu 20.04 LTS install. It also happened when upgrading an earlier version of decko on Ubuntu 18.04 LTS.

**Comment by ethn** [comment]
That gem/version is available at https://rubygems.org/gems/card-mod-script

Have you overwritten your gem sources somewhere, like in $HOME/.gemrc?  You can check by running `gem sources`. For me that yields this:

```
*** CURRENT SOURCES ***

https://rubygems.org/
```

If you don't see rubygems in your sources, that's probably the issue.

**Comment by bertdeb** [comment]
I get the exact same response:

```
gem sources
*** CURRENT SOURCES ***

https://rubygems.org/
```

**Comment by ethn** [comment]
Oh, wow, ok, yikes.

The "good" news is that I'm able to reproduce this. Something is up with the dependency installations. The beauty of gems is we usually don't have to think about these things.

You could use `gem install card-mod-script`, but that will lead to another uninstalled dependency and another and another. So I wouldn't do that unless you have an urgent need.

However, what might work for you (while I try to figure this out) is to go to the existing version and run `bundle update`.  That's the pattern I've been using extensively without a problem, and it should update all the gems in one command.

**Comment by ethn** [comment]
I found a mod gem that had not been successfully pushed to rubygems. After fixing that it worked for me, and I'm hoping it will work for you now, too. BUT I had already installed a few of the other gems by hand, so I wasn't really starting with a clean slate.

Can you try `gem install ruby` again and let me know if it's any different now? If it still doesn't work for you I'll install a new ruby and see if I can reproduce the error again.

Thanks for bringing this to our attention and for your patience while we get it fixed!

**Comment by bertdeb** [comment]
```
gem install ruby
Fetching ruby-0.1.0.gem
Successfully installed ruby-0.1.0
Parsing documentation for ruby-0.1.0
Installing ri documentation for ruby-0.1.0
Done installing documentation for ruby after 0 seconds
1 gem installed
```

However, decko still fails:
```
gem install decko
ERROR:  Error installing decko:
	card-mod-ace_editor requires card-mod-script (= 0.11.0)
```

**Comment by ethn** [comment]
sorry, I meant `gem install decko`. Thanks for testing.

**Comment by ethn** [comment]
I installed a new ruby (2.7.2) and installed and ran `gem install decko` and I eventually ran into the same problem. Unfortunately I've got some pressing deadlines and can't debug this right now. (We're actually using a temporary solution right now where decko depends on tons of mods; this will soon change and you will have much more flexibility to choose your mods, but we didn't want to throw that out in a random point release).

In the meantime, this should help you get through the install:

```
gem install card-mod-content
gem install card-mod-history
gem install card-mod-permissions
gem install card-mod-virtual
gem install card-mod-list
gem install card-mod-script
gem install card-mod-style
gem install decko
```
